server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/WebAdminServer.java

```diff
@@ -95,6 +95,7 @@ public void start() {
             port = new Port(service.port());
             LOGGER.info("Web admin server started");
         }
+        return this;
```

Comment:
It's not specific to this PR.
But could you re-explain to me why this method is named ```start``` ? 
I don't see it starting anything.